atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/abstracttests/AbstractTimestampManagementTest.java

```diff
@@ -28,6 +28,7 @@ public abstract class AbstractTimestampManagementTest {
     private final EteTimestampResource timestampClient =
             EteExtension.createClientToSingleNode(EteTimestampResource.class);
 
+    @SuppressWarnings("ImpossibleNullComparison")
     @Test
     public void shouldBeAbleToFetchAndForwardTimestamp() {
         assertThat(timestampClient.getFreshTimestamp()).isNotNull();
```

Comment:
`getFreshTimestamp` returns a  primitive, so this assert can never fail.

We should remove the useless assert instead of suppressing the warning. The same applies to the suppressions below.

Reply:
I guess the point of this was we're able to (i.e., I think the intention here is `assertThatCode(...).doesNotThrowAnyException()`). But yeah, `isNotNull()` isn't the best choice here